packages/apps/edge/express/src/init.ts

```diff
@@ -0,0 +1,163 @@
+import * as crypto from "crypto";
```

Comment:
looks like things have diverged here compaired to CF i will make sure these stay in sync